pybikes/bicimad.py

```diff
@@ -7,10 +7,15 @@
 import json
 
 from pybikes import BikeShareSystem, BikeShareStation, PyBikesScraper
+from pybikes.contrib import TSTCache
 
 
+PASSKEY = 'C3D0E659D8D397782B414AB6FCC477B5C727435FE91E72069D34CEBB2C1491B3B8563FDAC043EA704660C0E87E6FE503C39D38FF43F7447563B1E437B349ACEC'
+CLIENTID = '8ff527f9-f85b-45ef-b1b2-bd9eb59e0fff'
```

Comment:
Let's not commit these secrets

Reply:
I implemented it more like https://github.com/eskerda/pybikes/blob/master/pybikes/ecobici_ba.py, which seemed simpler and more fitting for this

Reply:
The "leak" is not a security issue as those are the default public credentials for the app. Also, my goal with this PR is that the citibik.es API gets updated, how can I inform that those are valid credentials so they can use them in prod?

Reply:
It is not a security issue, but they could fill a take down request on GitHub if they wanted. Not committing secrets, even when these are public, reduces our legal surface

Reply:
> The "leak" is not a security issue as those are the default public credentials for the app. Also, my goal with this PR is that the citibik.es API gets updated, how can I inform that those are valid credentials so they can use them in prod?

You can send them by email